assets/js/components/notifications/GoogleTagIDMismatchNotification.js

```diff
@@ -190,7 +190,7 @@ export default function GoogleTagIDMismatchNotification() {
 				description={ sprintf(
 					/* translators: 1: Currently GA4 property name. 2: Current GA4 property ID. 3: Newly linked GA4 property name. 4: Newly linked GA4 property ID. */
 					__(
-						'The Google Tag on your site is no longer associated with your current Google Analytics 4 property "%1$s (%2$s)". It is now recording metrics to another Google Analytics 4 property "%3$s (%4$s)". If you want to continue seeing Analytics data in the Site Kit dashboard, we suggest you switch Site Kit’s Google Analytics configuration to show data for this property.',
+						'The Google Tag on your site is no longer associated with your current Google Analytics 4 property "%1$s (%2$s)". It is now recording metrics to another Google Analytics 4 property "%3$s (%4$s)". If you want to continue seeing Analytics data in the Site Kit dashboard, we suggest you update Site Kit\'s Google Analytics configuration to show data for the property used in your Google Tag.',
```

Comment:
We should always use apostrophes, not single-quotes, in places like this:

```suggestion
						'The Google Tag on your site is no longer associated with your current Google Analytics 4 property "%1$s (%2$s)". It is now recording metrics to another Google Analytics 4 property "%3$s (%4$s)". If you want to continue seeing Analytics data in the Site Kit dashboard, we suggest you update Site Kit’s Google Analytics configuration to show data for the property used in your Google Tag.',
```

It was previously that way, but a good rule of thumb is you shouldn't need to escape single-quotes in string, most of the time 🙂